sbt_common/finatra-storage/src/main/scala/uk/ac/wellcome/finatra/storage/DynamoClientModule.scala

```diff
@@ -34,18 +35,12 @@ object DynamoClientModule extends TwitterModule {
                         endpoint: String,
                         accessKey: String,
                         secretKey: String): AmazonDynamoDB = {
-    val standardClient = AmazonDynamoDBClientBuilder.standard
-    if (endpoint.isEmpty)
-      standardClient
-        .withRegion(awsConfig.region)
-        .build()
-    else
-      standardClient
-        .withCredentials(
-          new AWSStaticCredentialsProvider(
-            new BasicAWSCredentials(accessKey, secretKey)))
-        .withEndpointConfiguration(
-          new EndpointConfiguration(endpoint, awsConfig.region))
-        .build()
+
+    DynamoClientFactory.create(
```

Comment:
There’s no reason to go via a buildDynamoClient method in the module now; just call the factory directly.